bazel/repository_locations.bzl

```diff
@@ -206,6 +206,15 @@ DEPENDENCY_REPOSITORIES_SPEC = dict(
         urls = ["https://github.com/google/libprotobuf-mutator/archive/{version}.tar.gz"],
         use_category = ["test"],
     ),
+    com_github_google_tcmalloc = dict(
+        project_name = "tcmalloc",
+        project_url = "https://github.com/google/tcmalloc",
```

Comment:
Add date of https://github.com/google/tcmalloc/commit/d1311bf409db47c3441d3de6ea07d768c6551dec commit under this line
`#2020-09-16`

Reply:
Added the date